CMakeLists.txt

```diff
@@ -130,13 +130,20 @@ include_directories(
   lib/jest/include
   test/include
 )
+add_definitions(
+-D__STDC_CONSTANT_MACROS
+-D__STDC_LIMIT_MACROS
+)
 add_executable(${PROJECT_NAME}_config_test ${CMAKE_CURRENT_LIST_DIR}/test/src/config/main.cpp)
 target_link_libraries(${PROJECT_NAME}_config_test
-  ${PROJECT_NAME}_boost	
+  ${PROJECT_NAME}_boost
   ${LUA_LIBRARIES}
   ${CURSES_LIBRARY}
   ${ZLIB_LIBRARIES}
   ${CMAKE_THREAD_LIBS_INIT}
+  clang
+  clangTooling
+  ${LLVM_LIBS}
```

Comment:
We can probably replace the two clang bits here with `${CLANG_LIBS}`, which may fix the travis build.